jujugui/static/gui/src/app/app.js

```diff
@@ -1767,6 +1767,10 @@ YUI.add('juju-gui', function(Y) {
         }
         charmstore.getEntity(url.legacyPath(), callback);
       };
+      const getModelName = () => {
+        const name = this.db.environment.get('name');
+        return name === 'untitled-model' ? undefined : name;
```

Comment:
I thought the db was the source of truth, but you're correct that the `environmentName` is. Updating